statistics/feedback.go

```diff
@@ -298,7 +301,7 @@ func buildBucketFeedback(h *Histogram, feedback *QueryFeedback) (map[int]*Bucket
 	for _, fb := range feedback.feedback {
 		skip, err := fb.adjustFeedbackBoundaries(sc, &min, &max)
 		if err != nil {
-			log.Debugf("adjust feedback boundaries failed, err: %v", errors.ErrorStack(err))
+			logutil.Logger(context.Background()).Debug("adjust feedback boundaries failed", zap.Error(err))
```

Comment:
@lamxTyler Should we change its level to ERROR?

Reply:
@lamxTyler PTAL